.github/workflows/update-from-template.yml

```diff
@@ -2,7 +2,7 @@ name: actions-template-sync
 
 on:
   schedule:
-  - cron:  "0 1 * * *"
+    - cron:  "0 1 * * *"
```

Comment:
FYI: [https://crontab.guru](https://crontab.guru/#0_1_*_*_*)
In case you didn't know about this side yet.

Reply:
This comes from the template repo we should change this there